src/Utilities/DeclarationBlockParser.php

```diff
@@ -82,7 +82,11 @@ public function parse(string $declarationBlock): array
         $properties = [];
         foreach ($declarations as $declaration) {
             $matches = [];
-            if (preg_match('/^(-?+[a-zA-Z_][a-zA-Z_0-9\\-]++|--[a-zA-Z_0-9\\-]++)\\s*+:\\s*(.++)$/s', \trim($declaration), $matches) === 0) {
+            if (preg_match(
+                '/^(-?+[a-zA-Z_][a-zA-Z_0-9\\-]++|--[a-zA-Z_0-9\\-]++)\\s*+:\\s*(.++)$/s',
```

Comment:
I've just noticed that the first part of this regex should probably have `*+` not `++` so that it can match a single character...

Reply:
> I'll just create a PR for it now.

#1506